doc/js_tutorials/js_setup/js_nodejs/js_nodejs.markdown

```diff
@@ -226,120 +226,3 @@ These C++ sources use standard APIs to access the filesystem and the implementat
 In the browser, this filesystem is emulated in memory while in Node.js there's also the possibility of using the local filesystem directly. This is often preferable since there's no need of copy file's content in memory. This section explains how to do just that, this is, configuring emscripten so files are accessed directly from our local filesystem and relative paths match files relative to the current local directory as expected.
 
 ### The example ###
-
```

Comment:
The example corresponds to `Dealing with files` chapter.  We cannot just remove the code, it should be replaced by something meaningful.

Reply:
> The example corresponds to `Dealing with files` chapter. We cannot just remove the code, it should be replaced by something meaningful.

Correct. I tried to replace it with the DNN-based FaceDetectorYN. However, FaceDetectorYN isn't exposed to JS, therefore it has to rewrite the post-processing part in JS. This appears somewhat redundant, @WanliZhong suggests if FaceDetectorYN can be exposed.

Reply:
Replaced the Haar based face detection example with FaceDetectorYN.